templates/generic.py

```diff
@@ -1,6 +1,6 @@
 from copy import deepcopy as copy
 
-from button import ButtonTemplate
+from .button import ButtonTemplate
```

Comment:
It's mostly a python-3 thing http://stackoverflow.com/questions/12172791/changes-in-import-statement-python3